src/tennissimulator/match/Match.java

```diff
@@ -0,0 +1,84 @@
+package tennissimulator.match;
+
+import tennissimulator.player.Player;
+
+import java.util.Optional;
+
+public class Match {
+
+    private static final int SETS_TO_WIN_MATCH = 3;
+    private static final int POINTS_TO_WIN_SET_TIEBREAK = 7;
+    private static final int POINTS_TO_WIN_MATCH_TIEBREAK = 10;
+
+    private final Player firstPlayer;
+    private final Player secondPlayer;
+    private ScoreTable scoreTable;
```

Comment:
Can it be final?

Reply:
fixed, moved initialization into constructor.